src/views/Pages/Register/Register.js

```diff
@@ -0,0 +1,241 @@
+import React, { Component } from 'react';
+import { Button, Card, CardBody, Col, Container, Form, Input, InputGroup, InputGroupAddon, InputGroupText, Row } from 'reactstrap';
+import { Nav, NavItem, NavLink, TabContent, TabPane } from 'reactstrap';
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+import { withRouter } from 'react-router-dom';
+import { fetchUserData } from '../../../store/actions/user'
+class Register extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      firstname: '',
+      lastname: '',
+      phonenumber: '',
+      email: '',
+      gender: 'M',
+      country: 'Nigeria',
+      dateofbirth: '',
+      formId: 'customer',
+      company: '',
+      address: '',
+      password: '',
+      loading: false,
+      activeTab: ["1"]
+    }
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+  toggle(tab) {
+    this.setState({
+      activeTab: [tab]
+    });
+  }
+
+  form(formId) {
+    let optInp;
+    let u = '';
+    let f = '';
+    let { firstname, phonenumber, email, gender, country, lastname, dateofbirth, company, address } = this.state
+    // console.log(this.state)
+    let { loading, error, errorMessage, userData } = this.props.userData
+    error === 'true' && errorMessage !== '' ? (f = errorMessage) :
+      (userData.errorcount !== '0' ? (f = userData.status) : (f = ''))
+    loading === 'done' ? (u = 'Create Account') :
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [':' should be placed at the beginning of the line.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)